### DISCUSSION TITLE
Send transform data via OSC

### DISCUSSION THREAD

**Discussion by ElbyB** [Iam1337/extOSC#28] [Q&A]
Hi,

I'm new to Unity and I would like to send an object rotation data via OSC, is someone here can help me getting start? I seem to only find exemples that uses UI buttons and sliders...

Thanks!

**Comment by Iam1337** [comment]
Hi! Use Vector3 Informer component for send position. Quaternion Informer for send rotation. Same Vector3 Event and Quaternion Event components for receive.
